assets/locales/nl.json

```diff
@@ -21,20 +21,20 @@
     },
     "alphaWarning": {
       "demoPurpose": "Deze versie is alleen voor demonstratie-doeleinden.",
-      "notFullyImplemented": "Niet alle functionaliteit is geimplementeerd en getest",
+      "notFullyImplemented": "Nog niet alle functionaliteiten zijn geïmplementeerd en goedgekeurd door de benodigde autoriteiten",
       "review": "Test a.u.b de app en check de code. Rapporteer problemen, bugs en algemene feedback aan",
       "start": "ok, begin"
     },
     "confimationCode": {
-      "enterCode": "A.u.b. scan de QR code die je van jouw test hebt gekregen, of vul het ID van jouw test-resultaat in om je positieve resultaat te verifieren",
+      "enterCode": "Scan de QR code die je van jouw test hebt gekregen, of vul het ID van jouw test-resultaat in om je positieve resultaat te verifieren",
```

Comment:
"enterCode": "Scan de QR code die je van jouw test hebt gekregen, of vul het ID van jouw test-resultaat in",

Reply:
@wilgert What do you think? since you edited the same line

Reply:
ok